entries/c/coinzilla.com.json

```
@@ -0,0 +1,12 @@
+{
+  "Coinzilla": {
+    "domain": "coinzilla.com",
+    "tfa": [
+      "totp",
+      "email"
+    ],
+    "keywords": [
+      "cryptocurrencies"
```

This is rather a marketing website, right? Therefore, `communication` is (currently) the correct category.
```suggestion
      "communication"
```